CHANGELOG.md

```diff
@@ -4,10 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v1.0.0-beta5 - Unreleased] - TBD
+## [v1.0.0-beta.5 - Unreleased] - TBD
 
 ### Added
 
+- The Paging mechanism via a Link with rel `next` or `prev` as defined for Item Search can also be used
+  for the STAC - Features endpoint `/collections/{collection_id}/items`.
```

Comment:
Isn't this a PR to add it to Collections? Items always supported pagination, so I think this changelog entry and the PR title are confusing/wrong?

Reply:
Okay.

Additionally, aren't last and first relation types also defined?

Reply:
no, at least not in Part 1.

Reply:
Do we define them somewhere? I know a couple of implementations support them. I'd at least mention them somewhere.

Reply:
I added them. I also added text about how many data stores will not allow the implementation of the `prev` link